dev/build/windows/makecoq_mingw.sh

```diff
@@ -119,7 +126,11 @@ mkdir -p "$PREFIXCOQ/bin"
 mkdir -p "$PREFIXOCAML/bin"
 
 # This is required for building addons and plugins
+# This must be CFMT (/cygdrive/c/...) otherwise coquelicot 3.0.2 configure fails.
+# coquelicot uses which ${COQBIN}/coqc to check if coqc exists. This does not work with COQBIN in MFMT.
 export COQBIN=$RESULT_INSTALLDIR_CFMT/bin/
+# This must be MFMT (C:/) otherwise bignums 68a7a3d7e0b21985913a6c3ee12067f4c5ac4e20 fails
```

Comment:
I guess the comment is misleading. MFMT should be the default. We build MinGW binaries and these cannot handle cygwin (CFMT) paths. If you give something like /cygdrive/c/bla to a MinGW application, it has no idea this should be C:\bla. So it is expected that paths should be MFMT, otherwise they cannot be given to any of the executables we build (ocamlc, ocamlfind, coqc, ...) . What is more strange is if something has to be CFMT. This typically only happens if the path is added to $PATH or used in other situations where the : hurts. As it looks "where" in cygwin cannot handle MFMT paths.